commander/src/commands/transaction/create/vote.ts

```diff
@@ -30,37 +30,33 @@ import { flags as commonFlags } from '../../../utils/flags';
 import { getNetworkIdentifierWithInput } from '../../../utils/network_identifier';
 import { getPassphraseFromPrompt } from '../../../utils/reader';
 
+interface RawAssetVote {
+	readonly delegateAddress: string;
+	readonly amount: string;
+}
+
 const processInputs = (
 	nonce: string,
 	fee: string,
 	networkIdentifier: string,
-	votes: ReadonlyArray<string>,
-	unvotes: ReadonlyArray<string>,
+	votes: ReadonlyArray<RawAssetVote>,
 	passphrase?: string,
 ) =>
-	castVotes({
+	newCastVotes({
 		nonce,
 		fee,
 		networkIdentifier,
 		passphrase,
 		votes,
-		unvotes,
 	});
 
 interface Args {
 	readonly nonce: string;
 	readonly fee: string;
 }
 
-const processVotes = (votes: string) =>
-	votes
-		.replace(/\n/g, ',')
-		.split(',')
-		.filter(Boolean)
-		.map(vote => vote.trim());
-
-const getValidatedPublicKeys = (inputs: ReadonlyArray<string>) => {
-	validatePublicKeys(inputs);
+const validateAddresses = (inputs: ReadonlyArray<RawAssetVote>) => {
+	inputs.map(rawVoteAsset => validateAddress(rawVoteAsset.delegateAddress));
```

Comment:
since it's not `mapping`, it should be `for-loop`

Reply:
Changed to for-of